DEVELOPER.md

```diff
@@ -260,13 +260,13 @@ export KUBECONFIG="$(kind get kubeconfig-path --name=kuma)"
 2. Deploy `Control Plane` to [KIND](https://kind.sigs.k8s.io/docs/user/quick-start) (Kubernetes IN Docker):
 
 ```bash
-make start/control-plane/k8s
+make kind/deploy/kuma
```

Comment:
does this instruction work like described? or should I also run 
```
make images && make docker/save
```
?

Reply:
`kind/deploy/kuma` depends on `kind/load`, which in turn depends on `kind/load/control-plane` etc. Then al these targets depend on `image/kuma-cp` etc., which then builds the app and the docker images. The trick is that kind actually loads the images straight from the local docker registry and does not need the intermediate step of saving the images. So yep, that is all we need here. And if not, I will fix it.

Reply:
That's great! I forgot KIND uses the local docker registry.